### PR TITLE
[DOCS] Fix unreachable links in the document

### PR DESCRIPTION
**Pull request by sarutak**
## What changes were proposed in this pull request?

Recently, I found two unreachable links in the document and fixed them.
Because of small changes related to the document, I don't file this issue in JIRA but please suggest I should do it if you think it's needed.

## How was this patch tested?

Tested manually.

### CODE REVIEW

**Review by srowen — APPROVED** [review]

**Review by HyukjinKwon — APPROVED** [review]

File: docs/building-spark.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ should run continuous compilation (i.e. wait for changes). However, this has not
 extensively. A couple of gotchas to note:
 
 * it only scans the paths `src/main` and `src/test` (see
-[docs](http://scala-tools.org/mvnsites/maven-scala-plugin/usage_cc.html)), so it will only work
+[docs](http://davidb.github.io/scala-maven-plugin/example_compile.html)), so it will only work
```
Comment (HyukjinKwon) [inline docs/building-spark.md]: @sarutak, what do you think about using http://davidb.github.io/scala-maven-plugin/example_cc.html instead? (I [checked it](https://github.com/davidB/scala-maven-plugin/blob/2.9.1/src/site/xdoc/usage_cc.xml) although it looks not a big deal though as they have similar information).

Comment (sarutak) [reply docs/building-spark.md]: I confirmed [Internet Archive](https://web.archive.org/web/20160314050540/http://scala-tools.org/mvnsites/maven-scala-plugin/usage_cc.html) and I found the link you suggested is more proper. I'll modify it soon. Thanks.

**Comment by HyukjinKwon** [comment]
LGTM

**Comment by srowen** [comment]
Merged to master/2.2, and now that I look again here, realize the last tests technically didn't pass. As it's a doc change only that passed before, I can't see it will fail, but iwll keep an eye out. Oops.

**Pull request closed without merging**